app.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] While you fix your `.eslint` which is not running yet, kindly remove this linter disabling comment. Linter is supposed to assist you to stick to standard and industry practices, which will single you out of the crowd. :wink:

Reply:
Fixed.